rules/rest-api-design/rule.md

```diff
@@ -83,33 +87,40 @@ Common sense approaches include protecting your endpoints via short-lived access
 
 Enforce HTTPS for encrypted communication and consider OAuth for user authentication and authorization, protecting against unauthorized access.  
 
-## Tip \#7 - Keep cross-resource references simple
+## 7. Keep cross-resource references simple
 
 For APIs that reference multiple resources (e.g., `userId` in a post endpoint), keep relationships simple to prevent over-complicating endpoints.
-
 Provide clear references or IDs rather than nested data whenever possible to keep API responses readable and easy to follow.  
 
-For example, avoid using messy query patameters such as `api/products?user_id=123&product_id=321`. Instead, use clearly defined endpoints like `api/users/123/products/321`.
+::: greybox
+`api/products?user_id=123&product_id=321`
+:::
 
-## Extra tip - Rate limiting  
+::: bad
+Figure: Bad example - messy query parameter
+:::
 
-Implement rate limiting to control the number of requests per user within a time frame, protecting the API from abuse and ensuring fair resource usage.
+::: greybox
+`api/users/123/products/321`
+:::
 
-Provide appropriate status codes and messages (e.g., `HTTP 429`) when limits are reached to inform users clearly.
+::: good
+Figure: Good example - clearly defined endpoint
+:::
 
-## Extra tip - Caching
+## Tip - Rate limiting  
```

Comment:
Hi @sethdaily 

- [ ] 1. Can you make a H2 "Tip" and then make the subitems H3s?

This is to avoid the repetition of the word "Tip - " in 3 titles in a row